src/components/CustomDisciplineMenuItem.tsx

```diff
@@ -1,14 +1,22 @@
 import { StackNavigationProp } from '@react-navigation/stack'
-import React from 'react'
+import React, { ReactElement, useState } from 'react'
+import { Animated, View } from 'react-native'
+import { RectButton } from 'react-native-gesture-handler'
+import Swipeable from 'react-native-gesture-handler/Swipeable'
 import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
 import styled from 'styled-components/native'
 
+import { TrashBinIcon } from '../../assets/images'
 import labels from '../constants/labels.json'
 import { useLoadGroupInfo } from '../hooks/useLoadGroupInfo'
 import { RoutesParamsType } from '../navigation/NavigationTypes'
+import AsyncStorage from '../services/AsyncStorage'
+import ConfirmationModal from './ConfirmationModal'
 import Loading from './Loading'
 import MenuItem from './MenuItem'
 
+const widthOfTrashButton = wp('18%')
```

Comment:
```suggestion
const widthOfTrashButton = `${wp('18%')}px`
```
i get a missing unit warning

Reply:
done